docs/.vitepress/addons.ts

```diff
@@ -25,6 +25,76 @@ export const community: AddonInfo[] = [
     },
     repo: 'https://github.com/AlbertBrand/slidev-addon-tldraw',
   },
+  {
+    id: 'slidev-addon-sync',
+    name: 'slidev-addon-sync',
+    description: 'Sync component for Slidev static build that uses a SSE or WS server',
+    author: {
+      name: 'Tony Cabaye',
+      link: 'https://github.com/tonai',
+    },
+    repo: 'https://github.com/Smile-SA/slidev-addon-sync',
+  },
+  {
+    id: 'slidev-component-pager',
```

Comment:
Might be better to have a components category then? What do you think @KermanX?

Reply:
How about adding some tags/keywords to each addon? For example, `Component`/`Intergration`/`Tool`

Reply:
Just to add some context, the first add-ons I created were named `slidev-component-*` because I was working on them before we released the add-on feature in Slidev (released with v0.32 on May 20th).
The name "addon" was not official at that time, so I chose "component" and was too lazy to change the name later.
But yes, it can still be interesting to have tags :+1: